datastructures/pieces/Piece.h

```diff
@@ -11,6 +11,7 @@ enum Color {
 struct Position {
     char column;
     int row;
+
```

Comment:
I saw in one of your commits that you added a == operator to the Position struct, is this where it should be?

Reply:
Yeah, there was a problem so I deleted it temporarily. Now I added it again.